### PR TITLE
[Mobile]: Improving accessibility on Image Block (deselected)

### PR DESCRIPTION
**Pull request by etoledom**
This PR improves accessibility on the Image Block.

To test:
Please refer to the [gutenberg-mobile side PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/794).

### CODE REVIEW

File: packages/block-library/src/image/edit.native.js [hunk 1]
```diff
@@ -327,7 +327,14 @@ class ImageEdit extends React.Component {
 		const progress = this.state.progress * 100;
 
 		return (
-			<TouchableWithoutFeedback onPress={ this.onImagePressed } disabled={ ! isSelected }>
+			<TouchableWithoutFeedback
+				accessible={ ! isSelected }
+				accessibilityLabel={ __( 'Image block' ) + '.' + alt }
+				accessibilityRole={ 'button' }
+				accessibilityHint={ __( 'Doulbe tap to edit the image' ) }
```
Comment (Tug) [inline packages/block-library/src/image/edit.native.js]: `Doulbe` => `Double`

File: packages/block-editor/src/components/media-placeholder/index.native.js [hunk 2]
```diff
@@ -16,7 +16,12 @@ import styles from './styles.scss';
 
 function MediaPlaceholder( props ) {
 	return (
-		<TouchableWithoutFeedback onPress={ props.onMediaOptionsPressed }>
+		<TouchableWithoutFeedback
+			accessibilityLabel={ __( 'Image block. Empty ' ) }
```
Comment (Tug) [inline packages/block-editor/src/components/media-placeholder/index.native.js]: I guess we can reuse the 'Image block' string here. `Image block. Empty` as a whole does not make much sense for translators:
```
accessibilityLabel={ __( 'Image block' ) + '. ' + __( 'Empty' ) }
```

File: packages/block-library/src/image/edit.native.js [hunk 3]
```diff
@@ -327,7 +327,14 @@ class ImageEdit extends React.Component {
 		const progress = this.state.progress * 100;
 
 		return (
-			<TouchableWithoutFeedback onPress={ this.onImagePressed } disabled={ ! isSelected }>
+			<TouchableWithoutFeedback
+				accessible={ ! isSelected }
+				accessibilityLabel={ __( 'Image block' ) + '.' + alt }
```
Comment (Tug) [inline packages/block-library/src/image/edit.native.js]: It could use a space after the period:
```
__( 'Image block' ) + '. ' + alt
```

**Comment by etoledom** [comment]
Thank you for the review @Tug ! 
Added the requested changes. Nice catches :)

File: packages/block-library/src/nextpage/edit.native.js [hunk 4]
```diff
@@ -14,20 +14,21 @@ import { __ } from '@wordpress/i18n';
  */
 import styles from './editor.scss';
 
-export default function NextPageEdit( { attributes } ) {
-	const { customText = __( 'Page break' ) } = attributes;
-	// Setting the font here to keep the CSS linter happy, it was demanding a syntax
-	// that React Native wasn't able to handle (adding a fallback generic font family).
-	const textStyle = {
-		...styles[ 'block-library-nextpage__text' ],
-		fontFamily: 'System',
```
Comment (etoledom) [inline packages/block-library/src/nextpage/edit.native.js]: This was creating an issue that disabled scrolling with VoiceOver active.
I'm not sure the specific reason, but rolling back this change fixed the issue.

**Comment by etoledom** [comment]
I cherry-picked a commit to fix an issue scrolling with VoiceOver active: https://github.com/WordPress/gutenberg/pull/14713/commits/14652d67401060aeb3c20bdb4d4f0c0aa77df7dc.

That commit also brought ScreenReader improvements for the NextPage block.

@Tug this is ready for another check. Adding @pinarol as reviewer too.

File: packages/block-library/src/image/edit.native.js [hunk 5]
```diff
@@ -328,7 +328,14 @@ class ImageEdit extends React.Component {
 		const progress = this.state.progress * 100;
 
 		return (
-			<TouchableWithoutFeedback onPress={ this.onImagePressed } disabled={ ! isSelected }>
+			<TouchableWithoutFeedback
+				accessible={ ! isSelected }
+				accessibilityLabel={ __( 'Image block' ) + '. ' + alt }
```
Comment (pinarol) [inline packages/block-library/src/image/edit.native.js]: the dot here can be localized also

File: packages/block-editor/src/components/media-placeholder/index.native.js [hunk 6]
```diff
@@ -16,7 +16,12 @@ import styles from './styles.scss';
 
 function MediaPlaceholder( props ) {
 	return (
-		<TouchableWithoutFeedback onPress={ props.onMediaOptionsPressed }>
+		<TouchableWithoutFeedback
+			accessibilityLabel={ __( 'Image block' ) + '. ' + __( 'Empty' ) }
```
Comment (pinarol) [inline packages/block-editor/src/components/media-placeholder/index.native.js]: the dot here can be localized also

File: packages/block-library/src/nextpage/edit.native.js [hunk 7]
```diff
@@ -14,19 +14,20 @@ import { __ } from '@wordpress/i18n';
...
-	};
+	const accessibilityTitle = attributes.customText || '';
+	const accessibilityState = isSelected ? [ 'selected' ] : [];
 
 	return (
-		<View style={ styles[ 'block-library-nextpage__container' ] }>
+		<View
+			accessible
+			accessibilityLabel={ __( 'Page break block' ) + '. ' + accessibilityTitle }
```
Comment (pinarol) [inline packages/block-library/src/nextpage/edit.native.js]: the dot here can be localized also

**Comment by pinarol** [comment]
I've tried adding accesibility label for Image itself also, I think it felt better to be able to hear the image info independent from its selected state. 

```
                                    <ImageBackground
                                        accessible={ true }
                                        accessibilityLabel={ __( ‘Image’ ) + ‘. ’ + alt }
```

**Review by pinarol — APPROVED** [review]
Thanks! @etoledom 🎉 
I think it'd be ok for me to approve since localization will be very simple and the extra accessibility label will be a part of selected version's PR 

:shipit:

**Comment by etoledom** [comment]
Thank you!

**Pull request merged**